src/stt/src/SteinerTreeBuilder.cpp

```diff
@@ -76,21 +76,18 @@ Tree SteinerTreeBuilder::makeSteinerTree(odb::dbNet* net,
   int min_fanout = min_fanout_alpha_.first;
   int min_hpwl = min_hpwl_alpha_.first;
 
-  if (min_fanout > 0) {
-    if (net->getTermCount()-1 >= min_fanout) {
-      net_alpha = min_fanout_alpha_.second;
-    }
-  }
-
-  if (min_hpwl > 0) {
+  if (net_alpha_map_.find(net) != net_alpha_map_.end()) {
+    net_alpha = net_alpha_map_[net];
+  } else if (min_hpwl > 0) {
```

Comment:
I am so glad to see the std::pair is gone.
If you are testing against 0 why is the default value -1?
That is inconsistent. I think 0 is the right choice for both.